ui-modules/blueprint-composer/app/components/quick-fix/quick-fix.js

```diff
@@ -55,6 +55,31 @@ export function computeQuickFixes(blueprintService, allIssues) {
             computeQuickFixesForIssue(issue, issue.entity, blueprintService, v.quickFixes)
         });
     });
+
+    allIssues.warnings.byMessage = {};
+    Object.values(allIssues.warnings.byEntity).forEach(list => {
+        list.forEach(issue => {
+            let key = issue.group+":"+issue.ref+":"+issue.message;
+            let v = allIssues.warnings.byMessage[key];
+            if (!v) {
+                v = allIssues.warnings.byMessage[key] = {
+                    group: issue.group,
+                    ref: issue.ref,
+                    message: issue.message,
+                    issues: [],
+                    quickFixes: {},
+                };
+            }
+
+            let issueO = {
```

Comment:
Is it `issueObject`?

Reply:
This is a copy-paste from existing code, see more detailled explanation below @algairim